cantabular-import/scs.sh

```diff
@@ -75,12 +75,16 @@ cloneServices() {
     done
 }
 
-
 pull() {
     cd $DIR
     for repo in $(ls -d $DIR/*/); do
         cd ${repo}
-        git pull
+	if [ -d ".git" ]; then
+        	git pull
+	fi
+	if [ -f "go.mod" ]; then
+		go get -u  all
+	fi 
```

Comment:
I don't agree with this, as ideally I want to test my microservice changes against the latest from Sandbox and if I update the dependencies:
1. I may bring incompatible changes and break the service locally, making it harder to detect where is the issue
2. I will not be testing against a code copy of Sandbox

Why would you need this?

Reply:
With this change you will be updating the dependencies of all your local ONS repos, leaving them with Git staged changes which will be messy.

Probably there is another way of updating the required libs for a few repos.

Reply:
I did had this in mind and thus mentioned in the PR itself. I need this specific thing locally as there is a major change in cantabular 10.1 which is now available in dp-api-client-go. But found that there are packages that are not updated and are hindering my local run. The correct solution will be to updated all and push. 
So its better not to commit this.. but will use locally